src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -37,6 +44,11 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
     workspaceClient: WorkspaceClient;
     dataSourceManagement?: DataSourceManagementPluginSetup;
   }>();
+
+  const defaultWorkspaceFormValues: Partial<WorkspaceFormData> = {
```

Comment:
Shouldn't we move the color stuff to the useWorkspaceForm hook?

Reply:
Do you mean added it in the `useState` of color? I think the default value should be in workspace create page right?  User may confused if they didn't set the color but we display a default value in the workspace details page for legacy data.

Reply:
I see, makes sense.